### PR TITLE
Thesilence docs

### PR DESCRIPTION
**Pull request by therealsilence**
- Document recently added / undocumented Storm commands
- Add docs for raw pivot syntax
- Add appropriate hyperlinks from User Guide to autodocs for Storm Libraries, Storm Types
- General cleanup (typos, broken links, minor revisions)

### CODE REVIEW

File: docs/synapse/userguides/storm_adv_methods.ipynb [hunk 1]
```diff
@@ -974,6 +922,16 @@
     "# Close cortex because done\n",
     "_ = await core.fini()"
    ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. _Types: https://synapse.docs.vertex.link/en/latest/synapse/autodocs/stormtypes_prims.html\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_adv_methods.ipynb:930]: These should be relative links so they point to the current documentation build and not the latest build.

File: docs/synapse/userguides/storm_ref_cmd.ipynb [hunk 2]
```diff
@@ -1431,8 +1585,15 @@
     "model\n",
     "-----\n",
     "\n",
-    "Storm includes ``model.*`` commands (specifically ``model.edge.*``) that allow you work with lightweight (light) edges (see :ref:`gloss-edge-light`).\n",
+    "Storm includes ``model.*`` commands that allow you to work with model elements.\n",
+    "\n",
+    "``model.deprecated.*`` commands allow you to view model elements (forms or properties) that have been marked as \"deprecated\", determine whether your Cortex contains deprecated nodes / nodes with deprecated properties, and optionally lock / unlock those properties to prevent (or allow) continued use of deprecated model elements.\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_cmd.ipynb:1590]: continued creation of deprecated model elements.

File: docs/synapse/userguides/storm_adv_vars.ipynb [hunk 3]
```diff
@@ -1100,6 +1091,15 @@
     "# Close cortex because done\n",
     "_ = await core.fini()"
    ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. _Libraries: ../autodocs/stormtypes_libs.html\n",
+    ".. _Node: ../autodocs/stormtypes_prims.html#node\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_adv_vars.ipynb:1100]: Do we not have automatically generated header values you can link to by reference for these?

File: docs/synapse/userguides/storm_adv_methods.ipynb (hunk 1 above)
Comment (therealsilence) [reply docs/synapse/userguides/storm_adv_methods.ipynb:930]: Darn, thought I caught all of these...

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**